packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -339,19 +360,29 @@ export class MarketOperationUtils {
         nativeOrders: SignedOrder[],
         takerAmount: BigNumber,
         opts?: Partial<GetMarketOrdersOpts>,
-    ): Promise<OptimizerResult> {
-        const _opts = { ...DEFAULT_GET_MARKET_ORDERS_OPTS, ...opts };
-        const marketSideLiquidity = await this.getMarketSellLiquidityAsync(nativeOrders, takerAmount, _opts);
-        return this._generateOptimizedOrdersAsync(marketSideLiquidity, {
-            bridgeSlippage: _opts.bridgeSlippage,
-            maxFallbackSlippage: _opts.maxFallbackSlippage,
-            excludedSources: _opts.excludedSources,
-            feeSchedule: _opts.feeSchedule,
-            allowFallback: _opts.allowFallback,
-            shouldBatchBridgeOrders: _opts.shouldBatchBridgeOrders,
-            quoteRequestor: _opts.rfqt ? _opts.rfqt.quoteRequestor : undefined,
-            shouldGenerateQuoteReport: _opts.shouldGenerateQuoteReport,
+    ): Promise<OptimizerResultWithReport> {
+        const defaultOpts = { ...DEFAULT_GET_MARKET_ORDERS_OPTS, ...opts };
```

Comment:
Curious - why rename this `defaultOpts`?  I don't think it's an accurate name because this really is the `default options` merged with the `custom options`.  Would prefer to leave as `_opts` as that seems to be the pattern in this file

Reply:
Good point! I'm trying to keep the `_` prefix as private because that's what we have usually used it for. I know that I may have broken a convention here. Shall I roll-back to `_opts`?